### DISCUSSION TITLE
How i Judge wallet connect??

### DISCUSSION THREAD

**Discussion by matthew2099** [WalletConnect/WalletConnectUnity#221] [Q&A]
When I just In Initialize WalletConnect.
And I still don't connect. But ActiveSession has Topic
How i Judge wallet connect??

**Comment by skibitsky** [comment]
You can use `WalletConnect.Instance.IsConnected` to check if there is a connected wallet.

**Reply by matthew2099** [reply]
Still i don't connect Wallet, WalletConnect.Instance.IsConnected is true...

**Reply by skibitsky** [reply]
Please open an issue
https://github.com/WalletConnect/WalletConnectUnity/issues/new?assignees=&labels=bug&projects=&template=%F0%9F%90%9E-bug-report.md&title=
